.github/workflows/release.yml

```diff
@@ -176,15 +174,18 @@ jobs:
           echo "BUILD_COMMAND=./build.sh" >> $GITHUB_ENV;
           echo "TERRAFORM_DIRECTORY=sample-apps/python-aws-sdk-aiohttp-terraform" >> $GITHUB_ENV;
           echo "EXPECTED_TRACE_TEMPLATE=adot/utils/expected-templates/python.json" >> $GITHUB_ENV;
+        # FIXME: (enowell) You can only Smoke Test 1 Sample App with this design. We will want to test
+        # multiple in the future (i.e. Go/.NET Sample Apps with the same Collector Layer).
+        # ToDo: Add .NET layer lambda instrumentation smoke test
```

Comment:
```suggestion
        # TODO: Test both .NET app & Go app in the same collector-only Lambda Layer Smoke Test
```

Reply:
> Thanks for these changes! Made some comments, most important is probably our strategy for `soaking.yml`. I think we should replace .NET as we did for `release.yml`.
> 
> Also did you mean to update the `opentelemetry-lambda` submodule? I think we should do that separately like we did in #136 so I will make that PR so we can merge it in first if that's okay with you?

Yeah I'm okay with merging the opentelemetry-lambda submodule changes in another PR. Feel free to add me as an reviewer. Thanks!

Reply:
Thanks! Added you to https://github.com/aws-observability/aws-otel-lambda/pull/162